### ISSUE TITLE
Migrate Hydrus object druid:gb129dk1428 to H2

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/happy-heron#2157]
This item was inadvertently missed in the original migration. 
It belongs to collection druid:tn582bs2355 / work:42.

Please be sure this does not get assigned a DOI during the migration. I don't think the migration scripts handle DOIs in any way, so I don't think this should be a problem.

**Comment by jcoyne** [comment]
The migration script we previously used was run on Hydrus.  Now that Hydrus is not around we may need to write a new migration script.

**Comment by vivnwong** [comment]
Where did we save the un-migrated objects?

**Comment by andrewjbtw** [comment]
Everything is in Fedora + preservation. The migration was really making copies of data + metadata for H2 to operate on independently of the rest of SDR. It was more like an import into H2 rather than a migration.

**Comment by amyehodge** [comment]
@andrewjbtw This is the object of Kate Barron's with a catkey. I seem to recall we decided that we actually shouldn't migrate this, I think because the presence of the catkey will prevent Kate from being able to edit it in H2. Am I right that we shouldn't migrate this? If so, I'll close this ticket.

**Comment by andrewjbtw** [comment]
Right. Previously this was edited repeatedly in both Hydrus and Argo but we can't go back and forth between the systems (for now, anyway).
